src/wp-includes/deprecated.php

```diff
@@ -6460,4 +6460,31 @@ function wp_add_editor_classic_theme_styles( $editor_settings ) {
 	array_unshift( $editor_settings['styles'], $classic_theme_styles_settings );
 
 	return $editor_settings;
-}
\ No newline at end of file
+}
+
+/**
+ * Prints a CSS rule to fix potential visual issues with images using `sizes=auto`.
+ *
+ * This rule overrides the similar rule in the default user agent stylesheet, to avoid images that use e.g.
+ * `width: auto` or `width: fit-content` to appear smaller.
+ *
+ * @since 6.7.1
+ * @deprecated 6.9.0 Use wp_enqueue_img_auto_sizes_contain_css_fix() instead.
```

Comment:
Per below, if we want to target the next minor release:

```suggestion
 * @deprecated 6.8.2 Use wp_enqueue_img_auto_sizes_contain_css_fix() instead.
```